src/Microsoft.Android.Templates/android-wear/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0">
+	<uses-feature android:name="android.hardware.type.watch" />
+	<uses-permission android:name="android.permission.WAKE_LOCK" />
+	<application android:allowBackup="true" android:icon="@mipmap/appicon" android:label="@string/app_name" android:supportsRtl="true" android:theme="@android:style/Theme.DeviceDefault">
```

Comment:
Does this also need `android:roundIcon="@mipmap/appicon_round"`?

Like we have here:

https://github.com/xamarin/xamarin-android/blob/6b43af37bc57b7eea2a213ee15de037e61784a1c/src/Microsoft.Android.Templates/android/AndroidManifest.xml#L3

Would it also be worth sharing the icons in a single folder? And we can make the build pull them in for both templates?

Reply:
I think it uses the same purple square `.png`, but a different `.xml`? Android then clips it to a circle.